index.html

```diff
@@ -216,11 +225,15 @@ <h5 class="text-white">Skills</h5>
             </div>
         </section>
       <form action="https://formspree.io/f/xlezlkyw" method="post">
-        <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="forms">  
+        <p class="mine">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+      <div class="inputs">
         <input name="full-name" type="text" placeholder="Full name" maxlength="30" required>
         <input name="email" type="email" placeholder="Email address" required>
         <textarea name="message" id="message" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
         <button class="get" type="submit">Get in touch</button>
+      </div>
+    </div>
         <hr class="line">
       </form>
```

Comment:
- [ ] As mentioned by the previous reviewer, you still didn't fix the form design to match Figma , kindly fix the design to match Figma (update the form inputs design: add padding and update borders and you should be fine)

- _**your design :**_ 

![image](https://user-images.githubusercontent.com/89282221/187039548-3d2b03a6-6278-4d4a-aec4-e01b33d7727e.png)

- **_Figma Design_**


![image](https://user-images.githubusercontent.com/89282221/187039663-e8c42063-cd9c-4099-bb7e-034657955641.png)